static/docs/commands-reference/fetch.md

```diff
@@ -196,8 +198,10 @@ by all DVC-files in the current branch, including for directories. The checksums
 `3863d0e317dee0a55c4e59d2ec0eef33` and `42c7025fc0edeb174069280d17add2d4`
 correspond to the `model.pkl` file and `data/features/` directory, respectively.
 
+Let's link files from local cache to the workspace with:
+
 ```dvc
-$ dvc checkout      <- links files from local cache to workspace
+$ dvc checkout
 Checking out '{'scheme': 'local', 'path': '.../example-get-started/model.pkl'}' with cache '3863d0e317dee0a55c4e59d2ec0eef33'.
```

Comment:
not for this PR - we should revisit and update output here

Reply:
Oh yeah. Lots of code samples need to be double checked... All maybe? Opened #551 for this discussion.